packages/flutter/test/widgets/parent_data_test.dart

```diff
@@ -276,12 +276,13 @@ void main() {
       exception.toString(),
       startsWith(
         'Incorrect use of ParentDataWidget.\n'
-        'The following ParentDataWidgets are providing parent data to the same RenderObject:\n'
+        'Competing ParentDataWidgets are providing parent data to the same RenderObject:\n'
         '- Positioned(left: 7.0, top: 6.0) (typically placed directly inside a Stack widget)\n'
         '- Positioned(top: 5.0, bottom: 8.0) (typically placed directly inside a Stack widget)\n'
-        'However, a RenderObject can only receive parent data from at most one ParentDataWidget.\n'
-        'Usually, this indicates that at least one of the offending ParentDataWidgets listed '
-        'above is not placed directly inside a compatible ancestor widget.\n'
+        'A RenderObject can receive parent data from more than one type of ParentDataWidget, '
+        'but not of the same conflicting type.\n'
+        'Alternatively, this can indicate that at least one of the offending ParentDataWidgets '
+        'listed above is not placed directly inside a compatible ancestor widget.\n'
```

Comment:
I am slightly disturbed by the word "Alternatively" here. To what is this an alternative?

Should this be reworded to something like:

> Usually, this indicates that at least one of the offending ParentDataWidgets listed above isn't placed inside a dedicated compatible ancestor widget that it isn't sharing with another ParentDataWidget of the same type.